tests/test_apply.py

```diff
@@ -875,12 +894,14 @@ def circuit():
             ("CRot", [math.pi / 2, 0, -math.pi], [-1 / 2, -1 / 2]),
             ("CRot", [0, math.pi / 2, -math.pi], [-1 / 2, 1 / 4]),
             ("CRot", [-math.pi, 0, math.pi / 2], [-1 / 2, -1 / 2]),
+            ("ControlledPhaseShift", [0], [-1 / 2, -1 / 2]),
+            ("ControlledPhaseShift", [-math.pi], [-1 / 2, -1 / 2]),
```

Comment:
I think another test or two are needed here. I would try to add some additional variety to the chosen angles (even if the outcome of the expval calculation doesn't change, which means it works as expected :slightly_smiling_face: ).

Reply:
Sure! I just added some more tests. I wrote the code using `pennylane` to compute expval and got the same answer for all variety of inputs (as expected of course).

Reply:
Moreover, for writing new test cases I firstly looked at the `pennylane/tests` to make sure that I hadn't missed any test cases for `ControlledPhaseShift`. Interestingly, there are tests running against the state `1/2|00>+sqrt(3)/2|11>` with two Z expvals for other gates except this gate at `./beta/test_default_tensor.py`.